Updated docs for `logging.apiLogging.obfuscateURLs` config option (#3122)
dapr/docs: daprdocs/content/en/operations/troubleshooting/api-logs-troubleshooting.md

    enabled: true
```

### Obfuscate URLs in HTTP API logging

> By default, logs for API calls in the HTTP endpoints include the full URL being invoked, for example `POST /v1.0/invoke/directory/method/user-123`, which could contain Personal Identifiable Information (PII).

Reviewer: ```suggestion
By default, logs for API calls in the HTTP endpoints include the full URL being invoked (for example, `POST /v1.0/invoke/directory/method/user-123`), which could contain Personal Identifiable Information (PII).
```

> To reduce the risk of PII being accidentally included in API logs (when enabled), Dapr can instead log the abstract route being invoked, for example `POST /v1.0/invoke/{id}/method/{method:*}`. This can help ensuring compliance with privacy regulations such as GDPR.

Reviewer: ```suggestion
To reduce the risk of PII being accidentally included in API logs (when enabled), Dapr can instead log the abstract route being invoked (for example, `POST /v1.0/invoke/{id}/method/{method:*}`). This ensures compliance with privacy regulations, such as GDPR.
```

Author: I am ok with most these changes except "can help ensuring" -> "ensures" because that could be interpreted as us giving legal advice :)

Author: Or to put it in another terms, we can't promise that doing that is enough to ensure compliance with GDPR. We can just say it will help

Reviewer: Ah good catch - ignore that edit then and keep "can help ensuring"

> To reduce the risk of PII being accidentally included in API logs (when enabled), Dapr can instead log the abstract route being invoked, for example `POST /v1.0/invoke/{id}/method/{method:*}`. This can help ensuring compliance with privacy regulations such as GDPR.

Author: ```suggestion
To reduce the risk of PII being accidentally included in API logs (when enabled), Dapr can instead log the abstract route being invoked (for example, `POST /v1.0/invoke/{id}/method/{method:*}`). This can help ensuring compliance with privacy regulations such as GDPR.
```

> To enable obfuscation of URLs in Dapr's HTTP API logs, set `logging.apiLogging.obfuscateURLs` to `true`, for example:

Reviewer: ```suggestion
To enable obfuscation of URLs in Dapr's HTTP API logs, set `logging.apiLogging.obfuscateURLs` to `true`. For example:
```

```yaml
logging:
  apiLogging:
    obfuscateURLs: true
```

> Note that logs emitted by the Dapr gRPC APIs are not impacted by this configuration option as they only include the name of the method invoked and no arguments.

Reviewer: ```suggestion
Logs emitted by the Dapr gRPC APIs are not impacted by this configuration option, as they only include the name of the method invoked and no arguments.
```
